backends/bmv2/bmv2.cpp

```diff
@@ -104,12 +104,11 @@ int main(int argc, char *const argv[]) {
     // Generate a PI control plane API for this program if requested.
     if (!options.p4RuntimeFile.isNullOrEmpty()) {
```

Comment:
Would you say it could be a good idea to move this code to a common location so that it doesn't have to be duplicated between backends (the code that looks at the command line options and invokes `generateP4Runtime` and the `serialize` methods?
This issue for example (https://github.com/p4lang/p4c/issues/935) adds for support for the p4runtime options in the eBPF backend and I think it would be a good idea to have support in the p4test backend as well.
All the backends would still need to call one method, but that's better than copying all these lines around. We also want backends to have to actively call a method anyway, as they may need to register hooks in the future to handle specific externs.

Reply:
Yeah, we could add a wrapper that would take care of this; this pattern is probably going to be the same in every backend. I anticipate those extern hooks being ultimately passed to `generateP4Runtime()`, but the wrapper could funnel them along to their destination.